example/README.md

```diff
@@ -11,6 +11,7 @@
    - `docker-compose run --rm job-api dm -u http://dmss:5000 reset ../app`
 5. Load test app data
     - `dm reset app`
+    - `dm import-plugin-blueprints node_modules/@development-framework/dm-core-plugins`
```

Comment:
can also add this to the oneliner command under Development tips

Reply:
fixed